templates/api/CMakeLists.txt.tpl

```diff
@@ -11,8 +11,43 @@ project({{ $lib_id }} LANGUAGES CXX)
 set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
-find_package(nlohmann_json REQUIRED)
-find_package(Qt6 REQUIRED COMPONENTS Core)
+{{- if not (or .Module.Interfaces .Module.Structs )}}
+set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
```

Comment:
Looking at the commit itself, you probably can remove this part completely.
This will also remove the unnecessary new line changes in the generated code.